src/client/cypress/e2e/detailPage/boreholeform.cy.js

```diff
@@ -158,4 +160,22 @@ describe("Test for the borehole form.", () => {
       ensureEditingEnabled();
     });
   });
+
+  it("Export a borehole.", () => {
+    const boreholeAlternateName = "AAA_HIPPOPOTHAMUS";
+    createBorehole({
+      "extended.original_name": "AAA_HIPPOPOTHAMUS",
+      "custom.alternate_name": boreholeAlternateName,
+    }).as("borehole_id");
+
+    deleteDownloadedFile(`${boreholeAlternateName}.json`);
+
+    cy.get("@borehole_id").then(id => {
+      goToRouteAndAcceptTerms(`/${id}`);
+      ensureEditingDisabled();
+      exportItem("detail-header");
```

Comment:
PP: du musst hier glaube ich nicht den Parent übergeben (bzw braucht es die Option vermutlich garnicht in der Funktion) die Option gibt es nur, weil es zt mehrere Buttons mit dem gleichen Inhalt gibt (zb Add in der Completion) und mit der Übergabe des Parents kann der richtige Button selektiert werden.